src/page/ShopRegistration/view/PC/index.tsx

```diff
@@ -150,8 +157,8 @@ export default function ShopRegistrationPC() {
                     onChange={saveImgFile}
                     ref={imgRef}
                   />
-                  {imgFile
-                    ? <img src={imgFile} className={styles['form__main-menu']} alt="" />
+                  {imageUrl
+                    ? <img src={imageUrl} className={styles['form__main-menu']} alt="" />
```

Comment:
alt를 비워두기보다는 간단히 설명추가해줘도 좋을것같아요.
```suggestion
                  {imageUrl
                    ? <img src={imageUrl} className={styles['form__main-menu']} alt="메인 메뉴" />
```

Reply:
alt에 `메인 메뉴` 설명 추가했습니다. 03f24d50c8546ad09b53f28417bcb2d9c6395094